_posts/2017-01-26-create-an-fsharp-project-in-vscode.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: post
+asset-type: post
+name: create-an-fsharp-project-in-vscode
+title: Create an F# project in VSCode
+date: 2017-01-26 10:00:00 +00:00
+author: Pedro Santos
+image:
+    src: /assets/img/custom/blog/2017-01-26-create-an-fsharp-project-in-vscode.jpg
+    attribution:
+      text: directions by Beat Tschanz
+      href: https://www.flickr.com/photos/btschanz/13793872333/
+tags:
+- F#
```

Comment:
Please use the tag `FSharp`, instead of  `F#` - otherwise Jekyll fails to create a tag